.github/CODEOWNERS

```diff
@@ -1,7 +1,44 @@
+# Default group
+*                                                       @Cray-HPE/docs-csm-reviewers
 
-*                       @Cray-HPE/docs-csm-reviewers
+# Metal team items.
+background/ncn_*                                        @Cray-HPE/metal
+install/*                                               @Cray-HPE/metal
+install/livecd                                          @Cray-HPE/metal
+operations/bare_metal                                   @Cray-HPE/metal
 
-background/ncn_*        @Cray-HPE/metal
-install/*               @Cray-HPE/metal
-install/livecd          @Cray-HPE/metal
-operations/network      @Cray-HPE/management-network
+# Continuous Integration Items
+Jenkinsfile.github                                      @Cray-HPE/CI
+
+# CMS Items
+operations/boot_orchestration                           @Cray-HPE/cms-core-bos
```

Comment:
Need to add write permission for all of these new groups to get rid of the errors above.

Reply:
Yes, I was waiting until this had approvals as to not add teams unnecessarily. After this has approvals I'll also make Backports

Reply:
Done.